lib/spack/spack/modules/common.py

```diff
@@ -615,7 +615,7 @@ def configure_options(self):
             try:
                 configure_args = getattr(pkg, attr)()
                 return ' '.join(configure_args)
-            except (AttributeError, IOError, KeyError):
+            except (AttributeError, IOError, KeyError, NameError):
```

Comment:
In which case you encountered `NameError` here?

Reply:
I don't know how this commit made it in my branch. Probably a bad rebase. Anyways. Removed.